.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!-- Please remove the appropriate section.
+For example, if this is a new feature, remove the "Bug Report" section -->
+
+# Bug Report
+
+Fixes issue #<NUMBER>
+
+## Summary
+<!-- Describe your fix, including an explanation of how to reproduce the bug
+before and after the PR.-->
+
+## Checklist
+- [ ] Signed all commits for DCO
+- [ ] Added tests
+- [ ] Updated documentation (as needed)
+- [ ] Updated migration guide (as needed)
+- [ ] `codecheck` passed (See
+  [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
+- [ ] All tests passed (See
+  [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
+- [ ] While waiting for a review on your PR, please help review
+[another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+archived%3Afalse+)
```

Comment:
Since SDFormat isn't in the `ignitionrobotics` org, maybe we could customize the link here just for SDF.

Reply:
What about including both? https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+

Reply:
Should we include both for all ignition libraries or just in SDFormat?

Reply:
Good question. When we do a 2nd pass on the other templates, we can include SDF links there too :+1:

Reply:
Done 04f46cc323e11a74271ece8dac1fc493280e4595